### DISCUSSION TITLE
Toggle to exclude Alpha / Beta versions of the components

### DISCUSSION THREAD

**Discussion by dashmesh** [DependencyTrack/dependency-track#971] [Ideas]
Hi Steve, 

I trust that you are doing great.

Is it possible to include a toggle to toggle-off showing warning for outdated components if the latest component is not a release candidate?

Thanks,
Dashmesh

**Comment by stevespringett** [comment]
I like that idea. I just don't know of a way to reliably do it that works across all ecosystems. If everything adopted semver, it would be really easy. That's not the reality. 

I would love to see a PR to support this on the server side that consisted of a lot of different types of real-world tests across all supported ecosystems. I'm in favor of supporting it. I just don't have the capacity to investigate what would be necessary to do so.

**Reply by walterdeboer** [reply]
I've created PR #2501 so only stable versions will be considered for outdated components. I don't see use for a toggle for unstable versions. These are only interesting (in dependency track at least) when no stable verison exists.

I'm also working on an endpoint for reporting outdated component findings (#2514) to partly fix #1833 and #2003.

**Reply by walterdeboer** [reply]
See PR https://github.com/DependencyTrack/dependency-track/pull/2521 for the outdated component findings endpoint
